database/index.js

```diff
@@ -11,14 +11,28 @@ db.once('open', function() {
   console.log("🎊 Mongoose is connected to server 🎊")
 });
 
+let featureSchema = mongoose.Schema({
+  heading: String,
+  description: String,
+  posX: Number,
+  posY: Number
+});
+
 let productSchema = mongoose.Schema({
   id: Number,
   name: String,
-  features: {header: String, features: Array},
+  product_features:
+    { header: String,
+      features: [featureSchema]
+    },
   related_products: Array,

```

Comment:
Wonder if another productSchema can be nested in the array recursively? I don't know if it can or what advantages it would provide for our project.